### ISSUE TITLE
Couldn't find jquery.ui.all

### ISSUE THREAD

**Issue by lcguida** [rajeevkannav/gutter#1]
Hi, I got the following error when trying to use gutter:

```
Started GET "/gutter" for 127.0.0.1 at 2014-07-23 17:37:26 -0300
Processing by Gutter::GutterController#index as HTML
  Rendered /home/leandro/.rvm/gems/ruby-2.1.1@rails3/gems/gutter-1.0.0/app/views/gutter/gutter/index.html.erb within layouts/gutter/application (0.0ms)
Completed 500 Internal Server Error in 15.8ms

ActionView::Template::Error (couldn't find file 'jquery.ui.all'
  (in /home/leandro/.rvm/gems/ruby-2.1.1@rails3/gems/gutter-1.0.0/app/assets/javascripts/gutter/application.js:15)):
    15:       <%= stylesheet_link_tag "gutter/#{css}", media: "all" %>
    16:   <% end %>
    17:   <%= stylesheet_link_tag "gutter/application", :media => "all" %>
    18:   <%= javascript_include_tag "gutter/application" %>
    19: 
    20:   <%= csrf_meta_tags %>
    21: </head>
  sprockets (2.2.2) lib/sprockets/context.rb:102:in `resolve'
  sprockets (2.2.2) lib/sprockets/context.rb:142:in `require_asset'
  sprockets (2.2.2) lib/sprockets/directive_processor.rb:215:in `process_require_directive'
  sprockets (2.2.2) lib/sprockets/directive_processor.rb:165:in `block in process_directives'
  sprockets (2.2.2) lib/sprockets/directive_processor.rb:163:in `each'
  sprockets (2.2.2) lib/sprockets/directive_processor.rb:163:in `process_directives'
  sprockets (2.2.2) lib/sprockets/directive_processor.rb:97:in `evaluate'
  tilt (1.4.1) lib/tilt/template.rb:103:in `render'
  sprockets (2.2.2) lib/sprockets/context.rb:193:in `block in evaluate'
  sprockets (2.2.2) lib/sprockets/context.rb:190:in `each'
  sprockets (2.2.2) lib/sprockets/context.rb:190:in `evaluate'
  sprockets (2.2.2) lib/sprockets/processed_asset.rb:12:in `initialize'
  sprockets (2.2.2) lib/sprockets/base.rb:249:in `new'
  sprockets (2.2.2) lib/sprockets/base.rb:249:in `block in build_asset'
  sprockets (2.2.2) lib/sprockets/base.rb:270:in `circular_call_protection'
  sprockets (2.2.2) lib/sprockets/base.rb:248:in `build_asset'
  sprockets (2.2.2) lib/sprockets/index.rb:93:in `block in build_asset'
  sprockets (2.2.2) lib/sprockets/caching.rb:19:in `cache_asset'
  sprockets (2.2.2) lib/sprockets/index.rb:92:in `build_asset'
  sprockets (2.2.2) lib/sprockets/base.rb:169:in `find_asset'
  sprockets (2.2.2) lib/sprockets/index.rb:60:in `find_asset'
  sprockets (2.2.2) lib/sprockets/bundled_asset.rb:16:in `initialize'
  sprockets (2.2.2) lib/sprockets/base.rb:252:in `new'
  sprockets (2.2.2) lib/sprockets/base.rb:252:in `build_asset'
  sprockets (2.2.2) lib/sprockets/index.rb:93:in `block in build_asset'
  sprockets (2.2.2) lib/sprockets/caching.rb:19:in `cache_asset'
  sprockets (2.2.2) lib/sprockets/index.rb:92:in `build_asset'
  sprockets (2.2.2) lib/sprockets/base.rb:169:in `find_asset'
  sprockets (2.2.2) lib/sprockets/index.rb:60:in `find_asset'
  sprockets (2.2.2) lib/sprockets/environment.rb:78:in `find_asset'
  sprockets (2.2.2) lib/sprockets/base.rb:177:in `[]'
  actionpack (3.2.19) lib/sprockets/helpers/rails_helper.rb:126:in `asset_for'
  actionpack (3.2.19) lib/sprockets/helpers/rails_helper.rb:27:in `block in javascript_include_tag'
  actionpack (3.2.19) lib/sprockets/helpers/rails_helper.rb:26:in `collect'
  actionpack (3.2.19) lib/sprockets/helpers/rails_helper.rb:26:in `javascript_include_tag'
  gutter (1.0.0) app/views/layouts/gutter/application.html.erb:18:in `__home_leandro__rvm_gems_ruby_______rails__gems_gutter_______app_views_layouts_gutter_application_html_erb___1437922963570389319_28916360'
  actionpack (3.2.19) lib/action_view/template.rb:145:in `block in render'
  activesupport (3.2.19) lib/active_support/notifications.rb:125:in `instrument'
  actionpack (3.2.19) lib/action_view/template.rb:143:in `render'
  actionpack (3.2.19) lib/action_view/renderer/template_renderer.rb:59:in `render_with_layout'
  actionpack (3.2.19) lib/action_view/renderer/template_renderer.rb:45:in `render_template'
  actionpack (3.2.19) lib/action_view/renderer/template_renderer.rb:18:in `render'
  actionpack (3.2.19) lib/action_view/renderer/renderer.rb:36:in `render_template'
  actionpack (3.2.19) lib/action_view/renderer/renderer.rb:17:in `render'
  actionpack (3.2.19) lib/abstract_controller/rendering.rb:110:in `_render_template'
  actionpack (3.2.19) lib/action_controller/metal/streaming.rb:225:in `_render_template'
  actionpack (3.2.19) lib/abstract_controller/rendering.rb:103:in `render_to_body'
  actionpack (3.2.19) lib/action_controller/metal/renderers.rb:28:in `render_to_body'
  actionpack (3.2.19) lib/action_controller/metal/compatibility.rb:50:in `render_to_body'
  actionpack (3.2.19) lib/abstract_controller/rendering.rb:88:in `render'
  actionpack (3.2.19) lib/action_controller/metal/rendering.rb:16:in `render'
  actionpack (3.2.19) lib/action_controller/metal/instrumentation.rb:40:in `block (2 levels) in render'
  activesupport (3.2.19) lib/active_support/core_ext/benchmark.rb:5:in `block in ms'
  /home/leandro/.rvm/rubies/ruby-2.1.1/lib/ruby/2.1.0/benchmark.rb:294:in `realtime'
  activesupport (3.2.19) lib/active_support/core_ext/benchmark.rb:5:in `ms'
  actionpack (3.2.19) lib/action_controller/metal/instrumentation.rb:40:in `block in render'
  actionpack (3.2.19) lib/action_controller/metal/instrumentation.rb:83:in `cleanup_view_runtime'
  activerecord (3.2.19) lib/active_record/railties/controller_runtime.rb:24:in `cleanup_view_runtime'
  actionpack (3.2.19) lib/action_controller/metal/instrumentation.rb:39:in `render'
  actionpack (3.2.19) lib/action_controller/metal/implicit_render.rb:10:in `default_render'
  actionpack (3.2.19) lib/action_controller/metal/implicit_render.rb:5:in `send_action'
  actionpack (3.2.19) lib/abstract_controller/base.rb:167:in `process_action'
  actionpack (3.2.19) lib/action_controller/metal/rendering.rb:10:in `process_action'
  actionpack (3.2.19) lib/abstract_controller/callbacks.rb:18:in `block in process_action'
  activesupport (3.2.19) lib/active_support/callbacks.rb:403:in `_run__4148180605095355821__process_action__1716216779366417066__callbacks'
  activesupport (3.2.19) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.19) lib/active_support/callbacks.rb:385:in `_run_process_action_callbacks'
  activesupport (3.2.19) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.19) lib/abstract_controller/callbacks.rb:17:in `process_action'
  actionpack (3.2.19) lib/action_controller/metal/rescue.rb:29:in `process_action'
  actionpack (3.2.19) lib/action_controller/metal/instrumentation.rb:30:in `block in process_action'
  activesupport (3.2.19) lib/active_support/notifications.rb:123:in `block in instrument'
  activesupport (3.2.19) lib/active_support/notifications/instrumenter.rb:20:in `instrument'
  activesupport (3.2.19) lib/active_support/notifications.rb:123:in `instrument'
  actionpack (3.2.19) lib/action_controller/metal/instrumentation.rb:29:in `process_action'
  actionpack (3.2.19) lib/action_controller/metal/params_wrapper.rb:207:in `process_action'
  activerecord (3.2.19) lib/active_record/railties/controller_runtime.rb:18:in `process_action'
  actionpack (3.2.19) lib/abstract_controller/base.rb:121:in `process'
  actionpack (3.2.19) lib/abstract_controller/rendering.rb:45:in `process'
  actionpack (3.2.19) lib/action_controller/metal.rb:203:in `dispatch'
  actionpack (3.2.19) lib/action_controller/metal/rack_delegation.rb:14:in `dispatch'
  actionpack (3.2.19) lib/action_controller/metal.rb:246:in `block in action'
  actionpack (3.2.19) lib/action_dispatch/routing/route_set.rb:73:in `call'
  actionpack (3.2.19) lib/action_dispatch/routing/route_set.rb:73:in `dispatch'
  actionpack (3.2.19) lib/action_dispatch/routing/route_set.rb:36:in `call'
  journey (1.0.4) lib/journey/router.rb:68:in `block in call'
  journey (1.0.4) lib/journey/router.rb:56:in `each'
  journey (1.0.4) lib/journey/router.rb:56:in `call'
  actionpack (3.2.19) lib/action_dispatch/routing/route_set.rb:608:in `call'
  railties (3.2.19) lib/rails/engine.rb:484:in `call'
  railties (3.2.19) lib/rails/railtie/configurable.rb:30:in `method_missing'
  journey (1.0.4) lib/journey/router.rb:68:in `block in call'
  journey (1.0.4) lib/journey/router.rb:56:in `each'
  journey (1.0.4) lib/journey/router.rb:56:in `call'
  actionpack (3.2.19) lib/action_dispatch/routing/route_set.rb:608:in `call'
  meta_request (0.3.3) lib/meta_request/middlewares/app_request_handler.rb:13:in `call'
  rack-contrib (1.1.0) lib/rack/contrib/response_headers.rb:17:in `call'
  meta_request (0.3.3) lib/meta_request/middlewares/headers.rb:16:in `call'
  meta_request (0.3.3) lib/meta_request/middlewares/meta_request_handler.rb:13:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/best_standards_support.rb:17:in `call'
  rack (1.4.5) lib/rack/etag.rb:23:in `call'
  rack (1.4.5) lib/rack/conditionalget.rb:25:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/head.rb:14:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/params_parser.rb:21:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/flash.rb:242:in `call'
  rack (1.4.5) lib/rack/session/abstract/id.rb:210:in `context'
  rack (1.4.5) lib/rack/session/abstract/id.rb:205:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/cookies.rb:341:in `call'
  activerecord (3.2.19) lib/active_record/query_cache.rb:64:in `call'
  activerecord (3.2.19) lib/active_record/connection_adapters/abstract/connection_pool.rb:479:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.19) lib/active_support/callbacks.rb:405:in `_run__2012863834447543736__call__3550816291145266174__callbacks'
  activesupport (3.2.19) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.19) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.19) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.19) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/reloader.rb:65:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.19) lib/rails/rack/logger.rb:32:in `call_app'
  railties (3.2.19) lib/rails/rack/logger.rb:16:in `block in call'
  activesupport (3.2.19) lib/active_support/tagged_logging.rb:22:in `tagged'
  railties (3.2.19) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.5) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.5) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.19) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.5) lib/rack/lock.rb:15:in `call'
  actionpack (3.2.19) lib/action_dispatch/middleware/static.rb:63:in `call'
  railties (3.2.19) lib/rails/engine.rb:484:in `call'
  railties (3.2.19) lib/rails/application.rb:231:in `call'
  rack (1.4.5) lib/rack/content_length.rb:14:in `call'
  railties (3.2.19) lib/rails/rack/log_tailer.rb:17:in `call'
  thin (1.6.2) lib/thin/connection.rb:86:in `block in pre_process'
  thin (1.6.2) lib/thin/connection.rb:84:in `catch'
  thin (1.6.2) lib/thin/connection.rb:84:in `pre_process'
  thin (1.6.2) lib/thin/connection.rb:53:in `process'
  thin (1.6.2) lib/thin/connection.rb:39:in `receive_data'
  eventmachine (1.0.3) lib/eventmachine.rb:187:in `run_machine'
  eventmachine (1.0.3) lib/eventmachine.rb:187:in `run'
  thin (1.6.2) lib/thin/backends/base.rb:73:in `start'
  thin (1.6.2) lib/thin/server.rb:162:in `start'
  rack (1.4.5) lib/rack/handler/thin.rb:13:in `run'
  rack (1.4.5) lib/rack/server.rb:268:in `start'
  railties (3.2.19) lib/rails/commands/server.rb:70:in `start'
  railties (3.2.19) lib/rails/commands.rb:55:in `block in <top (required)>'
  railties (3.2.19) lib/rails/commands.rb:50:in `tap'
  railties (3.2.19) lib/rails/commands.rb:50:in `<top (required)>'
  script/rails:6:in `require'
  script/rails:6:in `<main>'


  Rendered /home/leandro/.rvm/gems/ruby-2.1.1@rails3/gems/actionpack-3.2.19/lib/action_dispatch/middleware/templates/rescues/_trace.erb (0.9ms)
  Rendered /home/leandro/.rvm/gems/ruby-2.1.1@rails3/gems/actionpack-3.2.19/lib/action_dispatch/middleware/templates/rescues/_request_and_response.erb (0.7ms)
  Rendered /home/leandro/.rvm/gems/ruby-2.1.1@rails3/gems/actionpack-3.2.19/lib/action_dispatch/middleware/templates/rescues/template_error.erb within rescues/layout (4.9ms)
```

**Comment by johnleetran** [comment]
also having the issue 

couldn't find file 'jquery.ui.all'
  (in /Users/jtran00/.rvm/gems/ruby-2.0.0-p247/gems/gutter-1.0.0/app/assets/javascripts/gutter/application.js:15)

**Comment by dereksweet** [comment]
I resolved this issue by using version 4.2.1 of the jquery-ui-rails gem.. 

gem 'jquery-ui-rails', '~> 4.2.1'
